ml-agents/mlagents/trainers/ppo/multi_gpu_policy.py

```diff
@@ -180,7 +163,7 @@ def update(self, mini_batch, num_sequences):
         for batch, tower, reward_tower in zip(
             device_batches, self.towers, self.reward_signal_towers
         ):
-            feed_dict.update(self.construct_feed_dict(tower, batch, num_sequences))
+            # feed_dict.update(self.construct_feed_dict(tower, batch, num_sequences)) TODO: Fix multi-GPU optimizer
```

Comment:
If this is still todo, and multi-gpu does not work, maybe temporarily remove this class or mark it as "Not implemented" ?

Reply:
Temporarily removed multi-gpu, to be re-added soon.